vnext/ReactWindowsCore/Modules/PlatformConstantsModule.cpp

```diff
@@ -38,7 +38,7 @@ std::map<std::string, folly::dynamic> PlatformConstantsModule::getConstants() {
       // Since we're out-of-tree, we don't know the exact version of React Native
       // we're paired with. Provide something sane for now, and try to provide a
       // better source of truth later. Tracked by Issue #4073
```

Comment:
>4073 [](start = 57, length = 4)

I make a fix upstream that will let us use real version that we should get during the next upgrade.